tools/java/org/unicode/cldr/util/VoteResolver.java

```diff
@@ -105,18 +106,72 @@ public static Status fromString(String source) {
      */
     public static final int HIGH_BAR = Level.tc.votes;
 
+    /**
+     * Constants for vote counts. These are mostly private since only needed inside enum Level.
+     */
+    public static class VC {
+        private static final int ZERO = 0; // for both Level.locked and Level.anonymous
```

Comment:
This should be done as values in VoteResolver.Level rather than a separate set of integer constants. If you need an additional field, that can be added.

BTW, 
        private int votes;
        private int stlevel;
should be
        private final int votes;
        private final int stlevel;

Reply:
OK, I've refactored to avoid VC and to restore the hard-coded integers in the Level constructor. To avoid repeating hard-coded integers, the initialization of voteCountMenu needed to move outside the constructor. For the two constants that need to be public, I've changed VC.PERMANENT and VC.LOCKING to Level.PERMANENT_VOTES and Level.LOCKING_VOTES, respectively. I'll redo the PR shortly.